### ISSUE TITLE
Problem attaching 8 disk images

### ISSUE THREAD

**Issue by larsbrinkhoff** [PDP-10/klh10#26]
@rmaldersoniii reported a problem in PDP-10/its#588.

```
KLH10# ; Define basic KS10 device config - two RH11s each on its own Unibus
KLH10#
KLH10# devdef rh0 ub1 rh11 addr=776700 br=6 vec=254
KLH10# devdef rh1 ub3 rh11 addr=772440 br=6 vec=224
KLH10#
KLH10# ; Provide one disk, one tape in config ITS expects
KLH10#
KLH10# devdef dsk0 rh0.0 rp type=rp06 format=dlw8 path=../../out/its.0.rp06 iodly=0
KLH10# devdef dsk1 rh0.1 rp type=rp06 format=dlw8 path=../../out/its.1.rp06 iodly=0
KLH10# devdef dsk2 rh0.2 rp type=rp06 format=dlw8 path=../../out/its.2.rp06 iodly=0
KLH10# devdef dsk3 rh0.3 rp type=rp06 format=dlw8 path=../../out/its.3.rp06 iodly=0
KLH10# devdef dsk4 rh0.4 rp type=rp06 format=dlw8 path=../../out/its.4.rp06 iodly=0
KLH10# devdef dsk5 rh0.5 rp type=rp06 format=dlw8 path=../../out/its.5.rp06 iodly=0
KLH10# devdef dsk6 rh0.6 rp type=rp06 format=dlw8 path=../../out/its.6.rp06 iodly=0
KLH10# devdef dsk7 rh0.7 rp type=rp06 format=dlw8 path=../../out/its.7.rp06 iodly=0
KLH10#
KLH10# devdef mta0 rh1.0 tm02 fmtr=tm03 type=tu45
KLH10#
KLH10# ; ITS wants a 60Hz clock, allow it. Need this until defaults OK.
KLH10# set clk_ithzfix=60
clk_ithzfix: 60. => 60.
KLH10#
KLH10# ; Define IMP for PI on ITS.JOSS.COM
KLH10# devdef imp ub3 lhdh addr=767600 br=6 vec=250 ipaddr=172.16.36.247 gwaddr=172.16.36.1
Out of room for more device defs
```

**Comment by Rhialto** [comment]
The Makefile system contains an override -DKLH10_DEVMAX=12. Could that be the problem?

**Comment by larsbrinkhoff** [comment]
I see exactly 12 `devdef` in @rmaldersoniii's configuration.  I would have expected it to crash on number 13 if the limit was set to 12.  But close enough?  Maybe there's an internal device.

**Comment by larsbrinkhoff** [comment]
Confirmed.  Five disks are fine, adding one more triggers the `Out of room` message at `devdef` number 12.

**Comment by larsbrinkhoff** [comment]
Setting `-DKLH10_DEVMAX=13` allows six disks and bails at seven.  I'd say, we raise this limit to something roomy, yet reasonable.

**Comment by larsbrinkhoff** [comment]
Fixed by #28.
